WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1217,6 +1217,11 @@ ComposeViewHtmlEditor       = padding:@ComposeViewOutsidePadding@; background-co
 ComposeViewPlainTextEditor  = @FontSize-big@ @FontFamily-fixed@ @FullWidth@ @AppBorder@ overflow:auto; padding:8px;
 ComposeViewHeaderEmptyRight = padding-right: 18px;
 
+AttachBriefcaseItem         = border-style:solid; border-width:0 0 1px 0; border-color:transparent;
+AttachBriefcaseItemSelected = border-bottom-color:@PanelBorderColor@;
+AttachBriefcaseItemChild    = padding-left:@MailViewInnerSpacing@;
```

Comment:
If we don't need padding for other sides i.e 0px from all side expect left, can we have style like `padding:0 0 0 @MailViewInnerSpacing@`, to avoid the possibility of such styles applied accidentally by some other selectors. 

This is just suggestion, feel free to keep as it is, if you feels that won't be case and it's okay.

Thanks